randovania/games/prime2/patcher/csharp_subprocess.py

```diff
@@ -40,11 +46,13 @@ async def _read_data(stream: StreamReader, read_callback: Callable[[str], None])
             break
 
 
-async def _process_command_async(args: list[str], input_data: str, read_callback: Callable[[str], None]):
+async def _process_command_async(args: list[str], input_data: str, read_callback: Callable[[str], None],
+                                 envs: _Environ | dict = os.environ):
```

Comment:
Change this to be a `addition_path_entries: collections.abc.Sequence[str]`, then make this function responsible for creating the new env dict.